attributes/dotnet_agent.rb

```diff
@@ -1,8 +1,7 @@
-default['appdynamics']['dotnet_agent']['version'] = 'latest'
-default['appdynamics']['dotnet_agent']['source'] = 'http://localhost/AppdynamicsInstallers/' #'https://packages.appdynamics.com/machine/%{version}/JavaAppAgent.zip'
-default['appdynamics']['dotnet_agent']['checksum'] = '7acb3756147a1d5a13c49b107a890ea56a8eb4099fd793e498e34b6f0b5962dc' #nil
+default['appdynamics']['dotnet_agent']['version'] = '4.0.7.0'
```

Comment:
hey Dan, the reason I did this was because if you push latest like 4.1 and the controller doesn't support 4.1 thats going to be a problem right?